resolwe_bio/processes/workflows/bbduk_star_featurecounts_qc.py

```diff
@@ -0,0 +1,802 @@
+                )
+
+            class AlignmentOptions:
+                """Alignment and Seeding."""
+
+                align_overhang_min = IntegerField(
+                    label="Minimum overhang [--alignSJoverhangMin]",
+                    required=False,
+                    description="Minimum overhang (i.e. block size) for spliced alignments "
+                    "(default: 5).",
+                )
+                align_sjdb_overhang_min = IntegerField(
+                    label="Minimum overhang (sjdb) [--alignSJDBoverhangMin]",
+                    required=False,
+                    description="Minimum overhang (i.e. block size) for annotated (sjdb) spliced "
+                    "alignments (default: 3).",
+                )
+                align_intron_size_min = IntegerField(
+                    label="Minimum intron size [--alignIntronMin]",
+                    required=False,
+                    description="Minimum intron size: the genomic gap is considered an intron if its "
+                    "length >= alignIntronMin, otherwise it is considered Deletion (default: 21).",
+                )
+                align_intron_size_max = IntegerField(
+                    label="Maximum intron size [--alignIntronMax]",
+                    required=False,
+                    description="Maximum intron size, if 0, max intron size will be determined by "
+                    "(2pow(winBinNbits)*winAnchorDistNbins)(default: 0).",
+                )
+                align_gap_max = IntegerField(
+                    label="Minimum gap between mates [--alignMatesGapMax]",
+                    required=False,
+                    description="Maximum gap between two mates, if 0, max intron gap will be "
+                    "determined by (2pow(winBinNbits)*winAnchorDistNbins) (default: 0).",
+                )
+                align_end_alignment = StringField(
+                    label="Read ends alignment [--alignEndsType]",
+                    choices=[
+                        ("Local", "Local"),
+                        ("EndToEnd", "EndToEnd"),
+                        ("Extend5pOfRead1", "Extend5pOfRead1"),
+                        ("Extend5pOfReads12", "Extend5pOfReads12"),
+                    ],
+                    description="Type of read ends alignment (default: Local). Local: standard local "
+                    "alignment with soft-clipping allowed. EndToEnd: force end-to-end read alignment, "
+                    "do not soft-clip. Extend5pOfRead1: fully extend only the 5p of the read1, all "
+                    "other ends: local alignment. Extend5pOfReads12: fully extend only the 5' of the "
+                    "both read1 and read2, all other ends use local alignment.",
+                    default="Local",
+                )
+
+            class TwoPassOptions:
+                """Two-pass mapping options."""
+
+                two_pass_mode = BooleanField(
```

Comment:
This is defined but not used anywhere. STAR two-pass is most probably not relevant for this pipeline.